### DISCUSSION TITLE
How to configure the hostname?

### DISCUSSION THREAD

**Discussion by fergusonna** [openmiko/openmiko#93] [Q&A] [answered]
I'm running multiple cams, and I'm unsure what config files to create/change to control the hostname of each device.

**Comment by martinbutt** [comment]
I've tried to address this by adding dynamic DNS support (using [Inadyn](https://github.com/troglobit/inadyn)) in this [PR](https://github.com/openmiko/openmiko/pull/82). I can provide a binary with that module if you want to help test it? I have been using it for a while now with my cameras and a domain from Namecheap.

**Comment by robotgoat** [comment, marked as answer]
> I'm running multiple cams, and I'm unsure what config files to create/change to control the hostname of each device.

The hostname can be set by placing the file "hostname" in the overlay/etc/ directory on the sd card. Put whichever name you'd like on a single line and it will be applied.

**Reply by beankylla** [reply]
So
`Hostname cam1`
?

**Reply by robotgoat** [reply]
Sorry, I should have been clearer. Create the file named "hostname" in that directory. Do something like `touch hostname` and then on the first line of that file, state the name you want for the device. So, if you were to cat the file, the actual hostname will appear in your terminal. Alternatively you can use the command `echo cam1 > hostname` to do it in one step assuming you want to name the device cam1.
